server/src/main/scala/net/wiringbits/apis/EmailApi.scala

```diff
@@ -0,0 +1,51 @@
+package net.wiringbits.apis
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceClientBuilder
+import com.amazonaws.services.simpleemail.model.*
+import net.wiringbits.apis.models.EmailRequest
+import net.wiringbits.config.EmailConfig
+
+import javax.inject.Inject
+
+class EmailApi @Inject() (
```

Comment:
Let's create this:

```
trait EmailApi {
  def sendEmail(sender: String, recipient: String, body: String): Future[Unit]
}
```

You can have an AWS implementation like:

```
class EmailApiAWSImpl @Inject()(config: EmailConfig) extends EmailApi {...}
```

Use a module to wire this.